core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -880,18 +1036,303 @@ export class LoggedInUser extends BaseUser {
+      'Browse our Lessons',
+      'Creator Guidelines in the About Menu on navbar',
+      mathClassroomUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "Access the Android app" on the Teach page.
+   */
+  async clickAccessTheAndroidAppOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'Access the Android app',
+      'Creator Guidelines in the About Menu on navbar',
+      androidUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "VISIT CLASSROOM" on the Teach page.
+   */
+  async clickVisitClassroomOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'VISIT CLASSROOM',
+      'Creator Guidelines in the About Menu on navbar',
+      mathClassroomUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "BROWSE LIBRARY" on the Teach page.
+   */
+  async clickBrowseLibraryOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'BROWSE LIBRARY',
+      'Creator Guidelines in the About Menu on navbar',
+      communityLibraryUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link with the text "Explore Lessons" on the Teach page.
+   */
+  async clickExploreLessonsOnTeachPage(): Promise<void> {
+    await this.clickButtonToNavigateToNewPage(
+      'Explore lessons',
+      'Creator Guidelines in the About Menu on navbar',
+      mathClassroomUrl,
+      'Creator Guidelines'
+    );
+  }
+
+  /**
+   * Clicks the link on the Terms page about the CC-BY-SA 4.0 license.
+   */
+  async clickLinkToLicenseOnTermsPage(): Promise<void> {
+    await this.page.waitForXPath('(//a[contains(text(),"here")])[1]');
+    const [link] = await this.page.$x('(//a[contains(text(),"here")])[1]');
+    await Promise.all([this.page.waitForNavigation(), await link.click()]);
+
+    expect(this.page.url()).toBe(
+      'https://creativecommons.org/licenses/by-sa/4.0/legalcode'
+    );
+  }
+
+  /**
+   * Clicks the link on the Terms page that leads to the Oppia Announce google group.
+   */
+  async clickLinkToGoogleGroupOnTermsPage(): Promise<void> {
+    await this.page.waitForXPath('(//a[contains(text(),"here")])[2]');
+    const [link] = await this.page.$x('(//a[contains(text(),"here")])[2]');
+    await Promise.all([this.page.waitForNavigation(), await link.click()]);
+
+    expect(this.page.url()).toBe(googleGroupsOppiaAnnouceUrl);
+  }
+
+  /**
+   * Clicks the link on the Privacy Policy page that goes to the home page.
+   */
+  async clickLinkToHomePageOnPrivacyPolicyPage(): Promise<void> {
+    await this.page.waitForXPath(
+      '//a[contains(text(),"https://www.oppia.org")]'
+    );
+    await Promise.all([
+      this.page.waitForNavigation({waitUntil: 'networkidle0'}),
+      this.clickOn('https://www.oppia.org'),
+    ]);
+
+    expect(this.page.url()).toBe(learnerDashboardUrl);
```

Comment:
I tried logging in as a test user and clicked this button and it navigates to 'https://www.oppia.org'. It doesn't redirect to the Learner Dashboard link. Does the redirecting to the Learner Dashboard link happen to you?

Reply:
I think it redirects to the learner dashboard if you are signed in ... are you sure you are signed in?

Reply:
Yes, I'm sure I signed in and when I clicked the link. It navigates to 'https://www.oppia.org' instead of the learner dashboard. 
The redirecting to the learner dashboard only happens if the link is localhost:8181.

Reply:
Ah, thanks, I think I know what's happening, it's because the page has a direct link to www.oppia.org which is a different site for which the acceptance test is not logged in. So you are signed in to localhost but not to oppia.org.

I think it is OK for this to navigate to www.oppia.org and we treat it as an external link, if we hardcode the URL in the page. But if we change it to "/" then it would navigate to localhost:8181/learner-dashboard. Either way is OK from the perspective of the test, IMO.